ui-tests/src/test/java/io/syndesis/qe/steps/apiprovider/ApiProviderSteps.java

```diff
@@ -133,6 +135,7 @@ public void createTheTODOIntegration(String name, String source, String path) {
         cs.clickOnLink("Save");
         new CreateIntegrationSteps().setIntegrationName(name);
         cs.clickOnButton("Save");
+        Selenide.sleep(2000);
```

Comment:
https://selenide.org/javadoc/4.10/com/codeborne/selenide/Selenide.html
Selenide.sleep is not recommended. use `TestUtils.sleepWithoutInterrupt` or something like that